### DISCUSSION TITLE
Multi-block 1d spherical

### DISCUSSION THREAD

**Discussion by lanzam213** [idaholab/moose#21026] [Q&A Meshing] [answered]
Greetings, I have created a multi-block 1d spherical problem to solve transient heat transfer in a sphere encapsulated in 4 concentric shells. My question is if the spherical equation formulation is consistent with a 1d mesh geometry generated with CartesianMeshGenerator . I ask because the (X,Y,Z) coordinates of a Cartesian mesh suggests the spherical formulation could possibly not work, although a solution is achieved, and because I'm unable to open the exodus output file in ParaView. To create the mesh I used CartesianMeshGenerator to define 5 subdomains, as shown below. I appreciate any suggestions. 

[Mesh]
  [./mesh_gen]
    type = CartesianMeshGenerator
    dim = 1
    dx = '2.5e-4 9.5e-5 4.0e-5 3.5e-5 4.0e-5'
    ix = '29 11 5 4 5'
    subdomain_id = '1 2 3 4 5'
  [../]
[]

[./Problem]
  coord_type = RSPHERICAL
[../]

**Comment by GiudGiud** [comment, marked as answer]
Hello

Yes this should work. The mesh may look regular 1D but the code performs the necessary adjustments for volume / jacobian computations
You can convince yourself by solving a simple problem (diffusion?) and comparing to an analytical solution

Guillaume
